### PR TITLE
[IMP]  l10n : states data put in 'base'

### PR DESCRIPTION
**Pull request by sht-odoo**
task: https://www.odoo.com/web?#id=1967713&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720
pad: https://pad.odoo.com/p/r.75eaaf5a56643525833166ded74b1b8f

- states have been put in 'base'-> 'res.country.state.csv'
- According to new address format, uncessary states have been discarded from 'base' -> 'res.country.state.csv'.
- za (south africa), id (indonesia), ar (argentinia), uk (united kingdom), ro (romania), pl (poland) have been discarded from 'base' -> 'res.country.state.csv'.

### CODE REVIEW

**Comment by Yenthe666** [comment]
I'm not sure I get this. This'll drop all the states from the Netherlands that are defined in `res.country.state.csv`. So, where will they go? Will they be added in base itself again? CC @oco-odoo

**Comment by sht-odoo** [comment]
@Yenthe666 
As I have checked out new 'address format' for Netherlands , it does not use state data, that's why I have removed. 
http://www.upu.int/fileadmin/documentsFiles/activities/addressingUnit/nldEn.pdf
https://en.wikipedia.org/wiki/Address#Netherlands
I have taken reference from these two links and I have not found any state/province.

**Comment by Yenthe666** [comment]
@sht-odoo well maybe according to your links but I can tell you that they're still used in letters sometimes and in some cases you need them on letters.
For example: the cities 'Valkenburg' or 'Katwijk' exist at least two times in the Netherlands. Both of them are in different provinces, which means you would probably write the province on the letter to make sure it arrives well. 

I really do not see a single profit in removing them from Odoo either.

**Comment by sht-odoo** [comment]
@Yenthe666 ok then, I will keep states of Netherlands as it is in res.country.state.csv
Thanks for your suggestion and making correct this. :)

**Comment by Yenthe666** [comment]
Thanks a lot @sht-odoo, I really appreciate it!

**Comment by jva-odoo** [comment]
@jco-odoo we will keep states of Argentinian.
Related code is updated.

File: addons/l10n_do/__manifest__.py [hunk 1]
```diff
@@ -97,8 +97,7 @@
         'data/account_data.xml',
         'data/account_tax_report_data.xml',
         'data/account.tax.template.xml',
-        # Country States
-        'data/l10n_do_state_data.xml',
```
Comment (jco-odoo) [inline addons/l10n_do/__manifest__.py]: Not sure l10n_do_state_data.xml is removed in the branch

Comment (jco-odoo) [reply addons/l10n_do/__manifest__.py]: It could have been renamed as done in l10n_cr

Comment (jco-odoo) [reply addons/l10n_do/__manifest__.py]: @jva-odoo @sht-odoo @rgo-odoo

Comment (jva-odoo) [reply addons/l10n_do/__manifest__.py]: @jco-odoo We did not remove the file we rename it as you say. plz check here https://github.com/odoo/odoo/pull/32767/commits/7839a84d1fb5805699dd6f7f8ad687efef2fe2bb#diff-dfc36b852636aa3c0137ace250e5b66eR100

Comment (jva-odoo) [reply addons/l10n_do/__manifest__.py]: ok, you are right. forgot to remove 10n_do_state_data.xml from the data folder
@sht-odoo plz rename the file 10n_do_state_data.xml  -> l10n_do_res_partner_title.xml
 using git command.

Comment (jva-odoo) [reply addons/l10n_do/__manifest__.py]: @jco-odoo change done
Thank you

**Comment by jco-odoo** [comment]
@sht-odoo @jva-odoo Can you squash the 2 commits (in one nice message) and rebase so I can merge?

**Comment by jco-odoo** [comment]
@sht-odoo @jva-odoo Please also mention that it is not a problem to put all these states in base, because the current csv only takes 0.34 s to be loaded.

**Comment by jva-odoo** [comment]
@jco-odoo  ok @sht-odoo is working on it.

**Comment by jva-odoo** [comment]
@jco-odoo Done

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by Xavier-Do** [comment]
@robodoo retry

**Pull request closed without merging**